commit 4be3a4d2dcb3943f8dc344994b32a466045289b8 (refs/changes/47/3647/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-03-15T10:22:39-07:00 (1 year, 7 months ago)
    
    TRITON-245 sdc-sapi tweak 'jsl:' pragmas used for 'make check'

diff --git a/lib/server/audit.js b/lib/server/audit.js
index 2eb90fd..b37de98 100644
--- a/lib/server/audit.js
+++ b/lib/server/audit.js
@@ -64,9 +64,8 @@ function auditBodyFromReq(req, bodyOpts) {
 
     if (!bodyOpts.include) {
         auditBody = undefined;
-        /* jsl:ignore */
     } else if (!auditBody) {
-        /* jsl:end */
+        /* jsl:pass */
     } else if (Buffer.isBuffer(auditBody) && !bodyOpts.includeBuffers) {
         auditBody = '<buffer>';
     } else if (typeof (auditBody) === 'string' &&
@@ -107,9 +106,8 @@ function auditBodyFromRes(res, bodyOpts) {
     } else if (!bodyOpts.includeGet2xx && res.req.method === 'GET' &&
             (res.statusCode >= 200 && res.statusCode < 300)) {
         auditBody = undefined;
-        /* jsl:ignore */
     } else if (!auditBody) {
-        /* jsl:end */
+        /* jsl:pass */
     } else if (Buffer.isBuffer(auditBody) && !bodyOpts.includeBuffers) {
         auditBody = '<buffer>';
     } else if (typeof (auditBody) === 'string' &&
